RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -32,6 +32,7 @@
                                            'keep HFRecHitsSorted_hfrecoMB_*_*',
                                            #'keep ZDCDataFramesSorted_*Digis_*_*',
                                            'keep ZDCDataFramesSorted_hcalDigis_*_*',
+                                           'keep ZDCDataFramesSorted_castorDigis_*_*',
```

Comment:
I think that the same change should be added to RecoLocalCalo_EventContentCosmics_cff.py as well